### PR TITLE
[Bug 17577] Make sure we can set the hilitedItemName prop of the navbar widget

### PR DESCRIPTION
**Pull request by livecodepanos**

### CODE REVIEW

**Comment by livecodeali** [comment]
Probably worth adding a test for this too- check out https://github.com/livecode/livecode/blob/develop/extensions/widgets/header/tests/properties.livecodescript for examples with the header bar.

File: extensions/widgets/navbar/tests/properties.livecodescript [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+for more details.
+
+You should have received a copy of the GNU General Public License
+along with LiveCode.  If not see <http://www.gnu.org/licenses/>.  */
+
+on TestSetup
+   TestLoadExtension "com.livecode.library.widgetutils"
+   TestLoadExtension "com.livecode.library.iconSVG"
+   TestLoadExtension "com.livecode.widget.navBar"
```
Comment (livecodeali) [inline extensions/widgets/navbar/tests/properties.livecodescript]: I suspect the issue here is the capital b in navBar - these get used to build the paths to the extensions which are case-sensitive on linux.

Comment (livecodepanos) [inline extensions/widgets/navbar/tests/properties.livecodescript]: Thanks @livecodeali :)

**Comment by livecodeali** [comment]
@livecode-vulcan review ok 081e97c

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok 081e97cd1a422cbea8206ebedb73df34a66fddf2

**Comment by livecode-vulcan** [comment]
😎 test success 081e97cd1a422cbea8206ebedb73df34a66fddf2

* try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/1012)
* try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/1018)
* try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/1010)
* try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/1011)
* try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/1018)
* try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/1032)
* try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/1009)

**Pull request merged**